builtin/aws/ecs/components/platform/parameters.hcl

```diff
@@ -244,6 +244,13 @@ parameter {
   required    = false
 }
 
+parameter {
+  key         = "sidecar.container_ports"
+  description = "List of container port definitions for the container"
+  type        = "list of ecs.ContainerPort"
```

Comment:
Is there a way that we can inform the user what fields are needed to satisfy `ContainerPort`?